radicle-httpd/src/api/v1/projects.rs

```diff
@@ -378,6 +386,98 @@ async fn issues_handler(
     Ok::<_, Error>(Json(issues))
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct IssueCreation {
```

Comment:
```suggestion
pub struct IssueCreate {
```

Reply:
This has been removed, since I want to focus in this PR on authorize a session.